app/controls/open-settings.js

```diff
@@ -1,14 +1,19 @@
-var ipc = require('ipc')
-var BrowserWindow = require('browser-window')
-var path = require('path')
-var config = require('./../config')
-// var init = require('./../init')
+import ipc from 'electron-safe-ipc/host'
+import path from 'path'
+import config from './../config'
 
-ipc.on('open-settings', openSettings)
+const BrowserWindow = require('browser-window')
 
 function openSettings () {
-  var settingsWindow = new BrowserWindow(config.window)
+  const settingsWindow = new BrowserWindow(config.window)
+
+  if (process.env.NODE_ENV === 'production') {
+    settingsWindow.loadUrl('file://' + path.resolve(__dirname, '../settings.html'))
+  } else {
+    settingsWindow.loadUrl('http://localhost:3000/settings.html')
+  }
```

Comment:
Thanks for catching that @masylum , all that is config should go to the config.js file https://github.com/ipfs/electron-app/blob/master/app/config.js